src/bindings/python/flux/job/info.py

```diff
@@ -191,7 +191,7 @@ def __getattr__(self, attr):
             raise AttributeError
```

Comment:
In commit message for c4a3555c53a1ca7ce53b57c13b589590fdf4f233, missed a "t" in "AtributeError".